keiko/keiko/keiko.py

```diff
@@ -121,24 +121,22 @@ def generate_report(
             tex_output_file_name,
         ]
         env = {**os.environ, "TEXMFVAR": tmp_dirname}
-        output = subprocess.run(cmd, cwd=tmp_dirname, env=env, capture_output=True, check=False)
-        logger.info("pdflatex run. [report_id=%s] [template=%s] [command=%s]", report_id, template_name, " ".join(cmd))
-        logger.debug(output.stdout.decode("utf-8"))
-        logger.debug(output.stderr.decode("utf-8"))
-        if output.returncode:
-            raise Exception("Error running pdflatex")
-
-        output = subprocess.run(cmd, cwd=tmp_dirname, env=env, capture_output=True, check=False)
-        logger.info(
-            "pdflatex run. [report_id=%s] [template=%s] [command=%s]",
-            report_id,
-            template_name,
-            " ".join(cmd),
-        )
-        logger.debug(output.stdout.decode("utf-8"))
-        logger.debug(output.stderr.decode("utf-8"))
-        if output.returncode:
-            raise Exception("Error running pdflatex")
+        for i in enumerate(range(2), 1):
```

Comment:
```suggestion
        for i in range(1, 3):
```

Reply:
@ammar92 what do you think?

Reply:
Enumerate will give `(1,0)` and `(2, 1) ` and not `1` and `2`.

Reply:
> Enumerate will give `(1,0)` and `(2, 1) ` and not `1` and `2`.

Yep which is perfectly fine for: `for i, _ in enumerate(range(2)):`

Reply:
> Yep which is perfectly fine for: `for i, _ in enumerate(range(2)):`

It's just very verbose and not immediately clear what is meant. Maybe `for run in [1, 2]` is actually the clearest :smiley: . But the current `for i in enumerate(range(2), 1)` is wrong because that will print `(1, 0)` in the log message instead of `1`.

Reply:
> > Yep which is perfectly fine for: `for i, _ in enumerate(range(2)):`
> 
> It's just very verbose and not immediately clear what is meant. Maybe `for run in [1, 2]` is actually the clearest 😃 . But the current `for i in enumerate(range(2), 1)` is wrong because that will print `(1, 0)` in the log message instead of `1`.

Yes you're right, I should've written it right the first time (I forgot that it returned a tuple). I think your solution is the clearest.